docs/changelog-fragments/477.bugfix.rst

```diff
@@ -0,0 +1 @@
+- Fix missing includes under FreeBSD -- by :user:`donnerhacke`
```

Comment:
It's already dashed in Towncrier's Jinja2 template + it makes sense to use the past tense with more details here:
```suggestion
Extended CFLAGS to include header locations typical to FreeBSD
(:file:`/usr/local/include`) -- by :user:`donnerhacke`
```

Reply:
Thank you for the idea. Took it.